src/main/java/ch/njol/skript/expressions/ExprPlayerViewDistance.java

```diff
@@ -38,14 +39,15 @@
 @Name("View Distance")
 @Description("The view distance of a player. Can be changed.")
 @Examples({"set view distance of player to 10", "set {_view} to view distance of player",
-		"reset view distance of all players", "add 2 to view distance of player"})
-@RequiredPlugins("Paper 1.9-1.13.2")
+		"reset view distance of all players", "add 2 to view distance of player",
+		"NOTE: This may not work on some versions (such as MC 1.14.x). ",
+		"The return value in this case will be the view distance set in system.properties."})
```

Comment:
The examples should be formatted like this:
```java
@Examples({
		"set view distance of player to 10", "set {_view} to view distance of player",
		"reset view distance of all players", "add 2 to view distance of player",
		"NOTE: This may not work on some versions (such as MC 1.14.x). ",
		"The return value in this case will be the view distance set in system.properties."
})
```

Reply:
wait... why the heck did I put that note in examples? hahaha
I'm going to fix this all up